contracts/teller-network/Arbitrable.sol

```diff
@@ -58,6 +58,15 @@ contract Arbitrable {
         return arbitrationCases[_escrowId].open || arbitrationCases[_escrowId].result != ArbitrationResult.UNSOLVED;
     }
 
+    /**
+     * @notice Determine if a dispute existed for an escrow
+     * @param _escrowId Escrow to verify
+     * @return bool result
+     */
+    function hadDispute(uint _escrowId) public view returns (bool) {
+        return arbitrationCases[_escrowId].result != ArbitrationResult.UNSOLVED;
```

Comment:
Wouldn't this return true for escrows that don't have a dispute at all?

Reply:
No, because the default value for the `ArbitrationCase` struct `result` is `UNSOLVED`